client/app/scripts/components/node-details/node-details-health-link-item.js

```diff
@@ -23,7 +24,7 @@ export function appendTime(url, time) {
     const json = decodeURIComponent(url.substr(pos + cloudLinkPathEnd.length));
     try {
       payload = JSON.parse(json);
-      payload.time = { queryEnd: time.unix() };
+      payload.time = { queryEnd: moment(time).unix() };
```

Comment:
jsdoc for `time` type needs correction

Reply:
Where is that? :) It's the same format being seint through (unix seconds), it's just needs to be wrapped in `moment()` here, because `time` in now a string, not a moment object internally anymore.

Reply:
the jsdoc for this function still says Moment https://github.com/weaveworks/scope/blob/77b691ca0b67d91812934442077d57219cd750f0/client/app/scripts/components/node-details/node-details-health-link-item.js#L13